tests/unit/components/HdGallery.spec.js

```diff
@@ -0,0 +1,31 @@
+import { mount } from '@vue/test-utils';
+import HdGallery from '@/components/gallery/HdGallery.vue';
+import ITEMS from '@/stories/mocks/GALLERY_ITEMS';
+
+describe('HdGallery', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(HdGallery, {
+      propsData: {
+        items: ITEMS,
+      },
+    });
+  });
+
+  it('renders component', () => {
+    expect(wrapper.html()).toMatchSnapshot();
+  });
+
+  it('switchs elements and emits the proper events clicks', () => {
+    const testIndex = 3;
+    const figure = wrapper.find('.gallery__figure');
```

Comment:
We can declare the selector on top of the file to make it clear that this test depends on it (the same way we are doing it in other tests):
```js
const FIGURE_SELECTOR = '.gallery__figure';
```

Reply:
Done